### DISCUSSION TITLE
Error: Running LUA method 'mouseEvent'

### DISCUSSION THREAD

**Discussion by Aralysia** [Mogli12/KeyboardSteer#841] [Q&A]
Hello dear Moddevelopers, 

i encounterd an error in FS22 which is caused by this mod.
I'm playing with an Xbox 360 wireless Controller and this mod crashes my whole input of keyboard, mouse and controller if i try to couple something on my vehicle and when im going back into any menu.
Also i cant switch between different things attached to my vehicle.

My game is freshly installed and i have a couple of mods from the modhub, but my mates who play with keyboard and wheel didnt encounter any error, plus this mod worked a couple of months ago. 

My Error message says : 
Error: Running Lua method 'mouseEvent'.
dataS/scripts/input/InputBindingLua(3921): bad argument #1 to 'ipairs' (table expected, got nil)
![luaerror](https://user-images.githubusercontent.com/93770391/165169617-becc1833-670e-4039-a6dc-6b047b429ac7.png)
i hope someone can help me.

kind regards

**Comment by iSinterra** [comment]
It's just a key binding issue. If you have the FreezeTime mod installed, just change the toggle on/off key to something else.
